discovery_engine_core/web-api2/src/ingestion.rs

```diff
@@ -32,9 +33,11 @@ impl Application for Ingestion {
     type AppState = IngestionConfig;
 
     fn configure(config: &mut ServiceConfig) {
+        let healthcheck = web::resource("/health").route(web::get().to(HttpResponse::Ok));
         let resource = web::resource("/documents")
             .route(web::post().to(new_documents.error_with_request_id()));
-        config.service(resource);
+
+        config.service(healthcheck).service(resource);
```

Comment:
:art: ? As we always want to have this we could mount this once in `server.rs` (using `.service` just before the `.configure` call in the `App::new()` chain).